tests/cm_basic/test_shell.py

```diff
@@ -38,4 +38,4 @@ def test_001(self):
         HEADING("check if we can run help:return: ")
         r = run("whoami")
         print(r)
-        assert getpass.getuser() == r
+        assert getpass.getuser() in r
```

Comment:
This change is because the value given by `whoami` in a Windows enterprise environment includes the domain - `ADS\myuser` instead of the pure username and `getpass.getuser()` always returns the pure username.